doc/grattexDocumentation.Rnw

```diff
@@ -833,17 +833,76 @@ It may have mandatory or optional arguments, which occur immediately after \verb
 \end{tikzpicture}
 \end{figure}
 
-\section{Using \texttt{pandoc}}\label{sec:using-pandoc}
-\texttt{pandoc} comes installed with RStudio.
-To convert \texttt{orig.docx} to \texttt{new.tex}, run from the directory containing \texttt{orig.docx}.
-\begin{lstlisting}[language=bash]
-pandoc -s orig.docx -s --wrap=none --top-level-division=chapter -o new.tex
-\end{lstlisting}
 
 
+\section{Word document conversion using \texttt{pandoc}}\label{sec:using-pandoc}
+
+  You can use \texttt{pandoc} to convert Word documents to \LaTeX\ documents. It comes installed with RStudio and can be run from \texttt{terminal} (on Mac) or \texttt{cmd} (on Windows). If it hasn't been installed, you can install it at \url{https://pandoc.org/installing.html}. It requires a local version of \TeX. Converting a Word document to \LaTeX\ is a breezy three-part process.
+
+  \subsection{Preparing Word document}
+  There are some things \texttt{pandoc} should ignore (\eg~the Word Table of Contents, which will be automatically generated in \LaTeX), and some things it's not very good at (\eg~converting cross-references). To make the post-conversion tidying easier, prepare your Word document for conversion. Make a \textbf{copy} of your report, and:
+
+    \begin{enumerate}
+      \item Turn track changes off.
+      \item Remove the cover-page, table of contents, list of figures and bibliography.
+      \item Turn off numbering in all headers. Do this by selecting a \texttt{Heading 1}, removing the numbering, right-clicking on \texttt{Heading 1} in the \texttt{Style Pane} and selecting "Update Heading 1 to Match Selection". Repeat for Headings 2 and 3.
+      \item Turn off all hyperlinks. On Mac: select all, then \texttt{cmd+6}. On Windows: select all, then \texttt{ctrl+shift+F9}. Note that you will have to repeat this step for the footnotes (click on any footnote, select all, \dots).
+      \item Set the whole document to be single-column.
+      \item Save the Word document without spaces, \eg~\verb|mapping_converted.docx|.
```

Comment:
Save the Word document *using a filename* without spaces.

Reply:
Updated